### ISSUE TITLE
Deprecation warning upon finishing setup

### ISSUE THREAD

**Issue by Artim96** [Icinga/icingaweb2#5152]
## Describe the bug
When finishing setup, an error message appears together with the confirmation that setup has finished.

## To Reproduce
Provide a link to a live example, or an unambiguous set of steps to reproduce this issue. Include configuration, logs, etc. to reproduce, if relevant.

1. go through setup process until after the DB setup

## Expected behavior
Only the confirmation message appearing.

## Screenshots
![Screenshot from 2023-11-20 13-33-07](https://github.com/Icinga/icingaweb2/assets/16918678/5290bc14-d57a-49af-b762-30548fc9a524)

## Your Environment
Include as many relevant details about the environment you experienced the problem in

* Icinga Web 2 version and modules (System - About): 2.12.1-1+debian12
* Web browser used: Chrome
* Icinga 2 version used (`icinga2 --version`): r2.14.0-1
* PHP version used (`php --version`): 8.2.7
* Server operating system and version: Debian Testing

## Additional context
Error message doesn't seem to interfere with further setup.

**Comment by yhabteab** [comment]
Hii @Artim96, thanks for reporting!

This happens, though, if you don't provide LDAP binds `bind_dn`, `bind_pw` etc., you won't be able to log into Icinga Web after the wizard anyway without setting neither of them.

**Comment by Artim96** [comment]
They should not be needed as everything necessary is readable without. If they are still needed, is is a bug and I'd need to know how to add them after the fact. Just add "bind_dn=" and "bind_pw=" in the [icingaweb_ldap] section of the ressources.ini?

**Comment by yhabteab** [comment]
Didn't you skip this step? At least my setup does not find any users without setting the `bind_dn` and the ldap password.

<img width="927" alt="Bildschirmfoto 2023-11-20 um 16 38 13" src="https://github.com/Icinga/icingaweb2/assets/57616252/34b44011-e978-4fff-a7b6-707063eb1201">

**Comment by Artim96** [comment]
No, I didn't, as the setup was purposefully done in a way that those aren't needed, as it's only available through intrante.

**Comment by yhabteab** [comment]
I don't know how you successfully completed the wizard then, since these attributes are required, as you can see in the [docs](https://icinga.com/docs/icinga-web/latest/doc/04-Resources/#ldap).

**Comment by Artim96** [comment]
> I don't know how you successfully completed the wizard then, since these attributes are required, as you can see in the [docs](https://icinga.com/docs/icinga-web/latest/doc/04-Resources/#ldap).

Then you might want to tell that the installer. It wasn't required, it also passed validation without.

**Comment by yhabteab** [comment]
As you can see from the labels, I have clearly marked this as a bug. If you can log into your Icinga Web after the wizard, then you don't need to worry about this. I was just trying to point out how this deprecation error is triggered.

**Comment by Artim96** [comment]
> As you can see from the labels, I have clearly marked this as a bug. If you can log into your Icinga Web after the wizard, then you don't need to worry about this. I was just trying to point out how this deprecation error is triggered.

Actually I can't. But icingaweb isn't writing any logs, not even when setting log level to debug. Hence my question how to add a Bind DN and password after the fact.
